lookout/style/typos/utils.py

```diff
@@ -0,0 +1,177 @@
+from itertools import chain
+
+from gensim.models.fasttext import FastText
+import numpy
+import pandas
+
+
+def collect_embeddings(fasttext: FastText, tokens) -> numpy.ndarray:
+    return numpy.array([fasttext.wv[token] for token in tokens])
+
+
+def read_frequencies(file: str) -> dict:
+    """
+    Read token frequencies from the file.
+    :param file: Path to the file containing tokens with frequencies.
+                Every line has format "token count"
```

Comment:
Please align comments at the new line as in all other code - example https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/features.py#L222